capitulo 1/lang_en.json

```diff
@@ -3431,79 +3431,79 @@
   "obj_king_boss_slash_Step_0_gml_221_0": "\\E4* Mostre ao meu filho o MONSTRO que você é!/",
   "obj_king_boss_slash_Step_0_gml_223_0": "* .../",
   "obj_king_boss_slash_Step_0_gml_224_0": "\\E3* Tá^1, você quer ver o que eu posso fazer^1, é?/",
-  "obj_king_boss_slash_Step_0_gml_226_0": "* A vontade da Susie está mudando..^1.&* \\cYFALAR\\cW se tornou \\cYFOGO ESCARLATE\\cW!/%",
+  "obj_king_boss_slash_Step_0_gml_226_0": "* A força de vontade da Susie está mudando..^1.&* \\cYFALAR\\cW se tornou \\cYFOGO ESCARLATE\\cW!/%",
   "obj_king_boss_slash_Step_0_gml_232_0": "FogoEscarlat",
   "obj_king_boss_slash_Step_0_gml_233_0": "Dano#Escarlate",
   "obj_king_boss_slash_Step_0_gml_241_0": "* Sua ALMA brilhou seu poder para a Susie!/%",
   "obj_king_boss_slash_Step_0_gml_254_0": "* Ralsei tentou conversar com o REI.../",
   "obj_king_boss_slash_Step_0_gml_256_0": "* Você pode não ter percebido^1, mas.../",
   "obj_king_boss_slash_Step_0_gml_257_0": "\\E6* Esse é um mundo em que você não precisa lutar./",
   "obj_king_boss_slash_Step_0_gml_258_0": "\\E8* Eu sei que se nós tentarmos^1, conseguiremos achar outra maneira.../",
-  "obj_king_boss_slash_Step_0_gml_260_0": "\\E6* Pensamentos tão simples../",
+  "obj_king_boss_slash_Step_0_gml_260_0": "\\E6* Devaneios de uma mente unidimensional../",
   "obj_king_boss_slash_Step_0_gml_261_0": "\\E0* Uma pena que você não vai viver para perceber sua ingenuidade./",
   "obj_king_boss_slash_Step_0_gml_262_0": "\\E4* Agora^1, morra com os amigos que você ama!/",
   "obj_king_boss_slash_Step_0_gml_264_0": "* Desculpe^1, eu e meus amigos não vamos à lugar algum!/",
-  "obj_king_boss_slash_Step_0_gml_266_0": "* A vontade do Ralsei está mudando..^1.&* \\cYFALAR\\cW se tornou \\cYCURA TRIPLA\\cW!/%",
+  "obj_king_boss_slash_Step_0_gml_266_0": "* A força de vontade do Ralsei está mudando..^1.&* \\cYFALAR\\cW se tornou \\cYCURA TRIPLA\\cW!/%",
   "obj_king_boss_slash_Step_0_gml_273_0": "CuraTripla",
   "obj_king_boss_slash_Step_0_gml_274_0": "Cura#todos",
   "obj_king_boss_slash_Step_0_gml_282_0": "* Sua ALMA brilhou seu poder para o RALSEI!/%",
   "obj_king_boss_slash_Step_0_gml_25_0": "Caia perante&as correntes da&justiça!/%",
   "obj_king_boss_slash_Step_0_gml_27_0": "Como vocês &se atrevem&a vir aqui,&Iluminados..?/%",
-  "obj_king_boss_slash_Step_0_gml_28_0": "Vocês, isso&nos deixou&nas sombras,&despojados de&sentido.../%",
+  "obj_king_boss_slash_Step_0_gml_28_0": "Vocês nos&deixaram&nas sombras,&despojados de&sentido.../%",
   "obj_king_boss_slash_Step_0_gml_29_0": "Vocês OUSAM&voltar para&nos atormentar&mais uma vez?/%",
   "obj_king_boss_slash_Step_0_gml_30_0": "Saiam!&Encontramos&novos&propósitos./%",
-  "obj_king_boss_slash_Step_0_gml_31_0": "Para o&CAVALHEIRO&ter aparecido./%",
-  "obj_king_boss_slash_Step_0_gml_32_0": "O CAVALHEIRO&que extrai&as Fontes&da Terra./%",
+  "obj_king_boss_slash_Step_0_gml_31_0": "Desde que o&CAVALHEIRO&apareceu./%",
```

Comment:
CD e RMD

Reply:
Precisou de um textão aqui ._.
Cara, você viu que a frase tem um _has_? Se não viu sua mudança a princípio estaria errada, a não ser que seja a regra de uma coisa ter acontecido no passado e continua acontecendo no presente, o que em Inglês exige o verbo _have_ mas que em português dá pra resumir
Aliás. eu realmente estou ponderando que sua interpretação de _For the Knight has appeared_ na parte do _For the_ como se talvez tivesse outro significado nas regras da Língua Inglesa. Alguém aí sabe decor pra explicar?

Ah, mas tem um detalhe bem importante que faz com que eu tenha que negar sua mudança. O Rei está contando alguma história em vários conjuntos de frases em sequência, e essa linha e a próxima estão interligadas, mas essas mudanças que você quer fazer cortam a coerência entre elas. Portanto, continuam reprovadas as mudanças nessa linha.